exec.go

```diff
@@ -234,7 +234,6 @@ func getProcess(context *cli.Context, bundle string) (*specs.Process, error) {
 			p.Capabilities.Bounding = append(p.Capabilities.Bounding, c)
 			p.Capabilities.Effective = append(p.Capabilities.Effective, c)
 			p.Capabilities.Permitted = append(p.Capabilities.Permitted, c)
-			p.Capabilities.Ambient = append(p.Capabilities.Ambient, c)
```

Comment:
I understand that we should remove ambient if we don't set inheritable capabilities. But why we are not setting them? The commit that removed them doesn't say anything.

It will be great for the reason to remove them to be written down somewhere, at least in this commit message or PR description.

Reply:
Marking as solved, as you answer this already in a PR comment.